arq/worker.py

```diff
@@ -170,6 +175,8 @@ def __init__(
         self.poll_delay_s = to_seconds(poll_delay)
         self.max_tries = max_tries
         self.health_check_interval = to_seconds(health_check_interval)
+        if health_check_key is None:
+            self.health_check_key = self.queue_name + health_check_key_suffix
```

Comment:
```suggestion
            self.health_check_key = self.queue_name + health_check_key_suffix
        else:
            self.health_check_key = health_check_key
```
:-)

Reply:
also maybe you could add a test for this case.

Reply:
added test.